CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Backup] fix CR not deleting older backups [Forum#62783](https://xcp-ng.org/forum/post/62783) (PR [#6871](https://github.com/vatesfr/xen-orchestra/pull/6871))
```

Comment:
Shouldn't we use the new naming?

```suggestion
- [Delta Replication] Fix not deleting older replications [Forum#62783](https://xcp-ng.org/forum/post/62783) (PR [#6871](https://github.com/vatesfr/xen-orchestra/pull/6871))
```

Reply:
Also, if this leads to unexpected *full/key replications*, maybe that should be mentioned as well?

Reply:
Yes, and after re reading the code, the only other impact on backups are the metadata ( and I don't think the scheduleId is really used there)